### DISCUSSION TITLE
v1.1.1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2626] [Announcements]
## Windows App SDK 1.1.1 is now available

Windows App SDK 1.1.1 is a stable servicing release that fixes critical bugs for 1.1. 

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using the Windows App SDK 1.1 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

Check out our [release notes](https://aka.ms/windowsappsdk/1.1/1.1.1/release-notes) for the list of issues fixed in this release.

### Updating to this version

The 1.1.1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.1'>v1.1.1</a>.</em>

**Comment by shelllet** [comment]
![image](https://user-images.githubusercontent.com/16729017/173820916-bdbf1416-43ab-4303-9393-c1231964430d.png)

![image](https://user-images.githubusercontent.com/16729017/173820880-721da789-86eb-4e67-8c0d-81657d43a478.png)

why?

**Reply by shelllet** [reply]
may be my fault , when i add `<WindowsAppSDKSelfContained>true</WindowsAppSDKSelfContained>`, exception is missing.

**Reply by gabbybilka** [reply]
@BenJKuhn @btueffers can you help route? Thanks!

**Reply by BenJKuhn** [reply]
It looks like there's a version mismatch issue of some sort happening here. @DrusTheAxe or @sachintaMSFT should be able to help. @shelllet  can you open this as an issue for tracking?

**Reply by shelllet** [reply]
@BenJKuhn,  i already reference in new issue. #2682
